.jenkins/infrastructure/docker/dockerfiles/linux/base/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+#
+# Please use the associated build.sh to build this Dockerfile
+ARG UBUNTU_CODENAME
+FROM ubuntu:${UBUNTU_CODENAME}
```

Comment:
Does ubuntu have a minimal image? If so, that is preferable for running enclaves.